.github/workflows/continuous-integration.yml

```diff
@@ -44,9 +44,9 @@ jobs:
           chmod a+x builder.pyz
           AWS_CRT_SWIFT_CI_DIR="${{ env.AWS_CRT_SWIFT_CI_DIR }}" AWS_SDK_SWIFT_CI_DIR="${{ env.AWS_SDK_SWIFT_CI_DIR }}" SMITHY_SWIFT_CI_DIR="${{ env.SMITHY_SWIFT_CI_DIR }}" ./builder.pyz build -p ${{ env.PACKAGE_NAME }} --target=ios-armv8
   macos-compat:
-    runs-on: macos-11
+    runs-on: macos-12
     env:
-      DEVELOPER_DIR: /Applications/Xcode_13.2.1.app
+      DEVELOPER_DIR: /Applications/Xcode_14.0.app
```

Comment:
Won't this change the swift version this is run on? 
Do we want to continue to use Xcode 13.2.1 to force swift 5.5.x?

Reply:
I wanted to see if latest swift version solves the problem. Need to revert this back to original value. Don't want to this change as part of this PR as it is unrelated.